### PR TITLE
refactor: Update container/host names for services that do not use edgex-<service-key> format

### PR DESCRIPTION
**Pull request by lenny-intel**
**This PR is dependent on https://github.com/edgexfoundry/edgex-taf/pull/465 for Smoke Tests to pass.**

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/master/.github/CONTRIBUTING.md.

## What is the current behavior?
Some container/hostnames do not follow the naming standard of `edgex-<service-key>`

## What is the new behavior?
All container/hostnames now follow the naming standard of `edgex-<service-key>`

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: compose-builder/add-security.yml [hunk 1]
```diff
@@ -68,8 +68,8 @@ services:
   secretstore-setup:
     image: ${CORE_EDGEX_REPOSITORY}/security-secretstore-setup${ARCH}:${CORE_EDGEX_VERSION}${DEV}
     user: "root:root" # must run as root
-    container_name: edgex-secretstore-setup
-    hostname: edgex-secretstore-setup
+    container_name: edgex-security-secretstore-setup
+    hostname: edgex-security-secretstore-setup
```
Comment (jim-wang-intel) [inline compose-builder/add-security.yml:72]: are we going back to edgex-?  i remembered it was like that before

File: compose-builder/add-taf-app-services.yml [hunk 2]
```diff
@@ -42,15 +42,15 @@ services:
     image: ${APP_SVC_REPOSITORY}/app-service-configurable${ARCH}:${APP_SERVICE_VERSION}${APP_SVC_DEV}
     ports:
       - "59710:59703" #Exposing as different port to avoid conflict with other MQTT export instance
-    container_name: scalability-test-mqtt-export
-    hostname: scalability-test-mqtt-export
+    container_name: edgex-scalability-test-mqtt-export
+    hostname: edgex-scalability-test-mqtt-export
```
Comment (jim-wang-intel) [inline compose-builder/add-taf-app-services.yml:46]: use edgex-taf-scalability-test-mqtt-export?

**Review by jim-wang-intel — COMMENTED** [review]
i feel like we are going backward for the naming as before. what is main reason behind this?

File: compose-builder/add-security.yml (hunk 1 above)
Comment (lenny-intel) [reply compose-builder/add-security.yml:72]: We never change away for `edgex-` for container/hostname. This just change it to use the service key appropriately

File: compose-builder/add-taf-app-services.yml (hunk 2 above)
Comment (lenny-intel) [reply compose-builder/add-taf-app-services.yml:46]: No the service key is `scalability-test-mqtt-export`

File: compose-builder/add-security.yml (hunk 1 above)
Comment (jim-wang-intel) [reply compose-builder/add-security.yml:72]: ok got it.

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Comment by cherrycl** [comment]
recheck

**Comment by cloudxxx8** [comment]
it looks like recheck won't work until the conflicts resolved

**Review by jim-wang-intel — APPROVED** [review]

**Comment by lenny-intel** [comment]
recheck

**Comment by lenny-intel** [comment]
recheck

**Comment by lenny-intel** [comment]
recheck

**Pull request merged**